server/src/routes/global.ts

```diff
@@ -12,6 +13,17 @@ router.get("/911-calls", async (req: IRequest, res: Response) => {
   return res.json({ calls, status: "success" });
 });
 
+router.post("/911-calls", async (req: IRequest, res: Response) => {
+	const id = v4();
+	const { location, description, caller } = req.body
```

Comment:
Can you add a check if the description was not provided? The database will throw an error if there's no description.

```js
let description = req.body.description || "No description provided"
```